tests/itest_specs/basic_round_trip/short_call.cairo

```diff
@@ -1 +1,1033 @@
+        let (call_pool_balance_4) = ILiquidityPool.get_lpool_balance(
+            contract_address=amm_addr,
+            lptoken_address=lpt_call_addr
+        );
+        assert call_pool_balance_4 = 6955029451622672819;
+
+        let (put_pool_balance_4) = ILiquidityPool.get_lpool_balance(
+            contract_address=amm_addr,
+            lptoken_address=lpt_put_addr
+        );
+        assert put_pool_balance_4 = 11529215046068469760000;
+
+        // Test pool_locked_capital
+        let (call_pool_locked_capital_4) = ILiquidityPool.get_pool_locked_capital(
+            contract_address=amm_addr,
+            lptoken_address=lpt_call_addr
+        );
+        assert call_pool_locked_capital_4 = 0;
+
+        let (put_pool_locked_capital_4) = ILiquidityPool.get_pool_locked_capital(
+            contract_address=amm_addr,
+            lptoken_address=lpt_put_addr
+        );
+        assert put_pool_locked_capital_4 = 0;
+
+        // test value of pools position
+        let (pools_pos_val_call_5) = ILiquidityPool.get_value_of_pool_position(
+            contract_address = amm_addr,
+            lptoken_address = lpt_call_addr
+        );
+        assert pools_pos_val_call_5 = 0;
+        
+        let (pools_pos_val_put_5) = ILiquidityPool.get_value_of_pool_position(
+            contract_address = amm_addr,
+            lptoken_address = lpt_put_addr
+        );
+        assert pools_pos_val_put_5 = 0;
+
+        ///////////////////////////////////////////////////
+        // SETTLE BOUGHT OPTION
+        ///////////////////////////////////////////////////
+        IAMM.trade_settle(
+            contract_address=amm_addr,
+            option_type=0,
+            strike_price=strike_price,
+            maturity=expiry,
+            option_side=1,
+            option_size=half,
+            quote_token_address=myusd_addr,
+            base_token_address=myeth_addr
+        );
+
+        // Test balance of lp tokens in the account after the option was bought and after withdraw
+        let (bal_eth_lpt_5: Uint256) = ILPToken.balanceOf(
+            contract_address=lpt_call_addr,
+            account=admin_addr
+        );
+        assert bal_eth_lpt_5.low = 3000000000000000000;
+
+        let (bal_usd_lpt_5: Uint256) = ILPToken.balanceOf(
+            contract_address=lpt_put_addr,
+            account=admin_addr
+        );
+        assert bal_usd_lpt_5.low = 5000000000;
+
+        // Test amount of myUSD on option-buyer's account
+        let (admin_myUSD_balance_5: Uint256) = IERC20.balanceOf(
+            contract_address=myusd_addr,
+            account=admin_addr
+        );
+        assert admin_myUSD_balance_5.low = 5000000000;
+
+        // Test amount of myETH on option-buyer's account
+        let (admin_myETH_balance_5: Uint256) = IERC20.balanceOf(
+            contract_address=myeth_addr,
+            account=admin_addr
+        );
+        assert admin_myETH_balance_5.low = 6983736783540012627;
```

Comment:
btw, this means, that the user made cca 24USD loss... could make a not that explains where it came from?
`user made loss 0.016ETH (circa 24USD) loss that arose from option expiring in the money (25USD loss) and the diff of circa 1USD is because of received premia`